noxfile.py

```diff
@@ -7,12 +7,11 @@
     'git+https://github.com/GoogleCloudPlatform/python-repo-tools.git'
 
 DIRS = [
-    # Hello world doesn't have system tests, just a lint test which will be
```

Comment:
I take it you meant to delete this?

Reply:
I did mean to delete it... there is no hello world. this was for the previous step tutorial `1-hello-world`